ppx_version.opam

```diff
@@ -15,6 +15,8 @@ depends: [
   "ocaml" {<= "4.07.1"}
   "ppxlib"
   "ppx_bin_prot"
+  "ppx_deriving_yojson"
+  "ppx_sexp_conv"
```

Comment:
I think ideally we would want to add version constraints here : o but I don't think opam has a good story for that.